src/core/lib/viewport-transition-utils.js

```diff
@@ -1,162 +1,33 @@
-/* eslint max-statements: ["error", 50] */
-
-import {projectFlat, unprojectFlat} from 'viewport-mercator-project';
-import {Vector2, equals} from 'math.gl';
-import {mod} from '../math/utils';
 import assert from 'assert';
+import {mod} from '../math/utils';
 
-const EPSILON = 0.01;
+const VIEWPORT_PROPS =
+  ['width', 'height', 'longitude', 'latitude', 'zoom', 'bearing', 'pitch'];
 
-/** Util functions */
-function lerp(start, end, step) {
-  if (Array.isArray(start)) {
-    return start.map((element, index) => {
-      return lerp(element, end[index], step);
-    });
-  }
-  return step * end + (1 - step) * start;
-}
+const ANGULAR_PROPS = {
+  longitude: 1,
+  bearing: 1
+};
```

Comment:
no latitude and pitch?

Reply:
These are "wrapped angular props"

Reply:
Yes but shouldn't all angular props be wrapped?